addons/stock/models/stock_orderpoint.py

```diff
@@ -257,8 +258,20 @@ def _compute_qty_to_order(self):
                 remainder = orderpoint.qty_multiple > 0 and qty_to_order % orderpoint.qty_multiple or 0.0
                 if float_compare(remainder, 0.0, precision_rounding=rounding) > 0:
                     qty_to_order += orderpoint.qty_multiple - remainder
+
+            # Calculates the quantity that should be ordered to be a multiple of the PO UoM
+            min_qty_order_possible = orderpoint._get_min_qty_to_order()
+            if min_qty_order_possible:
+                qty_to_order = float_round(qty_to_order / min_qty_order_possible, precision_rounding=1, rounding_method='UP') * min_qty_order_possible
+
             orderpoint.qty_to_order = qty_to_order
 
+    def _get_min_qty_to_order(self):
+        """
```

Comment:
"""" text
"""
It's the standard we use for docstring. Not really intuitive but we do it that way